back-end/app.py

```diff
@@ -0,0 +1,107 @@
+## !/usr/bin/env python
+"""
+Author : Hwancheol Kang
+Project : Alpha-search
+Data : 5th, January, 2018
+Description : This script is for back-end service 
+including fulfillment webhook for an Dialogflow agent 
+and responding for web-client's request, Using flask
+"""
+from __future__ import print_function
+
+import json
+import os
+
+from flask import Flask
+from flask import request
+from flask import make_response
+
+import uuid
+import redis
+
+# Flask app should start in global layout
+app = Flask(__name__)
+
+# Generate Redis Object
+redis_obj = redis.Redis(host='localhost',port=6379,db=0)
+
+# This route is for fulfillment webhook for an Dialogflow agent.
+@app.route('/webhook', methods=['POST'])
+def webhook():
+    # Get json from dialogflow. 
+    req = request.get_json(silent=True, force=True)
+
+    print("Request:")
+    print(json.dumps(req, indent=4))
+
+    res = processWebhookRequest(req)
+
+    # JSON Encoding
+    res = json.dumps(res, indent=4)
```

Comment:
maybe we can create a constant (something like indent = 4) that holds the value 4 and use that constant for indents? In this case, it's easy to update when indent space count changes somehow :)